### ISSUE TITLE
Store Kits into JSON File

### ISSUE THREAD

**Issue by JustOneMoreBlock** [uEssentials/uEssentials#286]
Issue with Kits:
- The cooldown is great and only works if you don't restart your server. However, we do 2-hour restarts and resets the cooldown because it's stored in memory.

Proposal:
Save Kits to a JSON File. (Similar way of you storing warp data)

```
[
  {
    "SteamID": "00000000000000000",
    "Kits": [
      {
        "Name": "plusxp",
        "Created": "1463812079",
        "Expire": "1463819220"
      }
    ]
  }
]
```

The name of the kit is stored, so the database knows what the player has executed and stores the time of that command was executed in the "Created" file using Epoch Unix Time Stamp.

Using the same method as the name it will read from the kits.json (Needs to work on the Web as well.) and get the "Cooldown" and convert the seconds into hours and then use the "Created" timestamp to add +hours to create a new Epoch timestamp and store it as "Expire."

This will avoid using MySQL and SQLite for the time being and prevent the abusive behavior of kits.

This would be per Server so that you can store it in data as kitsdb.json or w/e. :)

**Comment by leonardosnt** [comment]
:+1:

**Comment by Daniil1288** [comment]
hooly crap this is going to change so much!! :D

**Comment by Daniil1288** [comment]
@JustOneMoreBlock pls check this out, won't be updating atm

**Comment by JustOneMoreBlock** [comment]
@leonardosnt @Daniil1288 

Tested it:
[http://www.imgwiz.com/images/2016/06/14/kQ18v.png](url)

data/kit_cooldowns.json
```
{
  "76561198066176197": {
    "Kits": {
      "plusxp": "2016-06-14T14:18:48.179409-04:00"
    },
    "Global": "0001-01-01T00:00:00"
  }
}
```

One caveat: I noticed it's still stored memory and upon saving and shutting down the server it will save to the file. However, if there was ever an issue of the server taking a dump possibly may not save to the file.

Possibly Solution:
Execute either after a /kit command has been executed or on a timer that uEssential uses. Possible Fix: We could probably even add in save on the "AutoCommands".
